source/geod24/concurrency.d

```diff
@@ -93,7 +63,7 @@ private
         bool doesIt = false;
         static foreach (T; Types)
         {
-            static if (is(T == Tid))
+            static if (is(T : Channel!R, R))
```

Comment:
Shouldn't we check `R` for local aliasing here ?

Reply:
Done